lib/ex_huobi/margin/websocket/market_ws.ex

```diff
@@ -117,7 +119,7 @@ defmodule ExHuobi.Margin.WebSocket.MarketWs do
 
       @impl true
       def handle_info(:subscribe_delta, state) do
-        topics = Map.get(state, :subscribe)
+        topics = Map.get(state, :subscribe) ++ Map.get(state, :bbo_subscribe)
```

Comment:
then here is just the list of topic. And you already loop through. Feel like you don't even need to have this PR